stdlib/runtime/bigint.gr

```diff
@@ -504,6 +504,32 @@ provide let toInt64 = (num: WasmI32) => {
   }
 }
 
+@unsafe
+provide let toUnsignedInt64 = (num: WasmI32) => {
+  let (<=) = WasmI64.leU
+  let (==) = WasmI32.eq
+  let numLimbs = getSize(num)
+  if (numLimbs == 0n) {
+    0N
+  } else if (numLimbs == 1n) {
+    let limb = getLimb(num, 0n)
+    limb
+    // TODO mayb
+    // if (limb <= _SMAX32_I64) {
+    //   if (flagIsSet(num, _IS_NEGATIVE)) {
+    //     WasmI64.mul(-1N, limb)
+    //   } else {
+    //     limb
+    //   }
+    // } else {
+    //     // throw Exception.NumberNotIntlike
+    //   throw Exception.Overflow
+    // }
```

Comment:
Delete? Make work? Add an issue with a TODO?

Reply:
Forgot to delete, mb